app/billing_client.go

```diff
@@ -0,0 +1,310 @@
+	}
+}
+
+// InvoiceSummary holds the information of an invoice summary.
+type InvoiceSummary struct {
+	ID            string
+	InvoiceDate   *timestamppb.Timestamp
+	InvoiceAmount float64
+	Status        string
+	DueDate       *timestamppb.Timestamp
+	PaidDate      *timestamppb.Timestamp
+}
+
+func invoiceSummaryFromProto(summary *pb.InvoiceSummary) *InvoiceSummary {
+	return &InvoiceSummary{
+		ID:            summary.Id,
+		InvoiceDate:   summary.InvoiceDate,
+		InvoiceAmount: summary.InvoiceAmount,
+		Status:        summary.Status,
+		DueDate:       summary.DueDate,
+		PaidDate:      summary.PaidDate,
+	}
+}
+
+// BillingClient is a gRPC client for method calls to the Billing API.
+type BillingClient struct {
+	client pb.BillingServiceClient
+}
+
+// NewBillingClient constructs a new BillingClient using the connection passed in by the Viam client.
+func NewBillingClient(conn rpc.ClientConn) *BillingClient {
+	return &BillingClient{client: pb.NewBillingServiceClient(conn)}
+}
+
+// GetCurrentMonthUsage gets the data usage information for the current month for an organization.
+func (c *BillingClient) GetCurrentMonthUsage(ctx context.Context, orgID string) (*GetCurrentMonthUsageResponse, error) {
+	resp, err := c.client.GetCurrentMonthUsage(ctx, &pb.GetCurrentMonthUsageRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return getCurrentMonthUsageResponseFromProto(resp), nil
+}
+
+// GetOrgBillingInformation gets the billing information of an organization.
+func (c *BillingClient) GetOrgBillingInformation(ctx context.Context, orgID string) (*GetOrgBillingInformationResponse, error) {
+	resp, err := c.client.GetOrgBillingInformation(ctx, &pb.GetOrgBillingInformationRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return getOrgBillingInformationResponseFromProto(resp), nil
+}
+
+// GetInvoicesSummary returns the outstanding balance and the invoice summaries of an organization.
+func (c *BillingClient) GetInvoicesSummary(ctx context.Context, orgID string) (float64, []*InvoiceSummary, error) {
+	resp, err := c.client.GetInvoicesSummary(ctx, &pb.GetInvoicesSummaryRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return 0, nil, err
+	}
+	var invoices []*InvoiceSummary
+	for _, invoice := range resp.Invoices {
+		invoices = append(invoices, invoiceSummaryFromProto(invoice))
+	}
+	return resp.OutstandingBalance, invoices, nil
+}
+
+// GetInvoicePDF gets the invoice PDF data.
+func (c *BillingClient) GetInvoicePDF(ctx context.Context, id, orgID string) ([]byte, error) {
+	stream, err := c.client.GetInvoicePdf(ctx, &pb.GetInvoicePdfRequest{
+		Id:    id,
+		OrgId: orgID,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	var data []byte
+	for {
+		resp, err := stream.Recv()
+		if err != nil {
+			if errors.Is(err, io.EOF) {
+				break
+			}
+			return data, err
+		}
+		data = append(data, resp.Chunk...)
+	}
+
+	return data, nil
+}
+
+// SendPaymentRequiredEmail sends an email about payment requirement.
```

Comment:
can you include how `customerOrgID` and `billingOwnerOrgID` are used in the function in the comment? I think it will make it a little more clear

Reply:
Hmm... I tried to look for the definition in the [proto definition](https://github.com/viamrobotics/api/blob/d7436a969cbc03bf7e84bb456b6dbfeb51f664d7/proto/viam/app/v1/billing.proto#L135-L138), and I can't seem to find it. It also doesn't exist in the other SDKs.